contracts/src/v0.8/keystone/CapabilityRegistry.sol

```diff
@@ -33,7 +33,9 @@ contract CapabilityRegistry is OwnerIsCreator, TypeAndVersionInterface {
     address signer;
   }
 
-  struct NodeCapability {
+  struct Node {
+    /// @notice The node's parameters
+    NodeParams params;
```

Comment:
Does this correctly batch `address signer` and `uint32 configCount`?

Reply:
Packed these into a single struct